src/transformers/models/vit/convert_vit_timm_to_pytorch.py

```diff
@@ -0,0 +1,1271 @@
+# coding=utf-8
+# Copyright 2020 The HuggingFace Inc. team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Convert ViT checkpoints from the timm library."""
+
+
+import argparse
+from pathlib import Path
+
+import torch
+import torchvision.transforms as T
```

Comment:
As in the feature extraction class, can we avoid the one letter import and jsut import the classes needed?